chainer_chemistry/functions/activation/softplus.py

```diff
@@ -0,0 +1,16 @@
+from chainer import functions
+
+
+def improved_softplus(x):
+    """
```

Comment:
where is the original source of this function, how did you take `absolute` value??

Reply:
Source : https://github.com/materialsvirtuallab/megnet/blob/f91773f0f3fa8402b494638af9ef2ed2807fcba7/megnet/activations.py#L6